src/jsfriendapi.rs

```diff
@@ -25,13 +25,38 @@ pub fn JS_WrapPropertyDescriptor(cx: *mut JSContext,
 
 pub mod bindgen {
     use jsapi::{JSContext, JSObject, JSClass, JSRuntime};
-    use libc::{uintptr_t, uint8_t, uint32_t};
+    use libc::{uintptr_t, int8_t, uint8_t, int16_t, uint16_6, int32_t, uint32_t, float32_t, float64_t};
```

Comment:
`uint16_6` should be `uint16_t`; `float32_t` and `float64_t` should be `c_float` and `c_double`, respectively.